### DISCUSSION TITLE
map not being saved after mapping pass (Dreame L10 Pro)

### DISCUSSION THREAD

**Discussion by fr4nk3n5731n** [Hypfer/Valetudo#1750] [Q&A / Support] [answered]
Hi there,

lately Valetudo doesn't seem to save the map after a mapping pass. I don't get the "There is a new map available, do you want to save?" notification either.
No matter how many times I run a mapping pass nothing happens.
I tried using Version 1105 and 1138 of the Dustcloud Firmware with various Valetudo Versions (2023.02.0 - 6b01ed5, 2023.01.0 - 3d6ef60, 2022.11.0 - 61aa741, 2022.09.0 - 8e3c889) but nothing worked.

I don't know when exactly it stopped working unfortunately.

Robot: Dreame L10 Pro
Firmware: 1138
Valetudo: 2023.02.0 (537f7af9203257c3f44125c27669c25386d6722a)

**Comment by Hypfer** [comment, marked as answer]
I assume you already did a factory reset?

Did you also try with a smaller map by e.g. locking it in a room?

**Reply by fr4nk3n5731n** [reply]
I was sure that I tried resetting it already, but I did it again for good measure now. It worked. Thank you.
